app/uk/gov/hmrc/lisaapi/controllers/AccountController.scala

```diff
@@ -38,357 +38,349 @@ class AccountController extends LisaController with LisaConstants {
 
   //region Create Or Transfer Account
```

Comment:
I think these region sections have become stale (apologies if it was because I did a code reformat).

Also, I wonder whether these are a hint that perhaps the account controller has become too big and it would benefit from us splitting out those "sections" if there isn't too much inter-dependent code (an initial glance doesn't look like much is shared).

Reply:
I think you're right, it'd probably benefit from being split out. Will come back to this after I've addressed all the other review comments.

Reply:
I've split these out now